README.md

```diff
@@ -96,7 +96,7 @@ rails db:create
 rails db:migrate
 ```
 
```

Comment:
![image](https://github.com/fithamlak/Budget-App/assets/93039359/3e69a3bf-778c-4023-a09d-07323e4ac2cc)

- [ ] It would be very nice if you re-record your video after making all the above changes because your video is exceeding the 5 mins limit 😨 Kindly make your video again in the range of 5 mins to follow the project requirements 👍